<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddInfo</class>
 <widget class="QWidget" name="AddInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局样式表*/
QWidget#Addfriend{
	background-color: rgb(255, 255, 255);
}

/*局部样式表top*/
QWidget#top_widget{
	border:1px solid rgb(149, 149, 149);
}

QPushButton#add_Friend{
	border-radius:5px;
	background-color: rgb(0, 170, 255);
	font:normal 18px &quot;楷体&quot;;
	color:rgb(255, 255, 255);
}

QPushButton#add_Friend:hover{
	background-color: rgb(0, 85, 255);
}

QPushButton#add_Group{
	border-radius:5px;
	background-color: rgb(0, 170, 255);
	font:normal 18px &quot;楷体&quot;;
	color:rgb(255, 255, 255);
}

QPushButton#add_Group:hover{
	background-color: rgb(0, 85, 255);
}

/*局部样式表middle*/
QPushButton#ok_Btn{
	border-radius:5px;
	background-color: rgb(0, 170, 255);
	font:normal 18px &quot;楷体&quot;;
	color:rgb(255, 255, 255);
}

QPushButton#ok_Btn:hover{
	background-color: rgb(0, 85, 255);
}

QWidget#middle_Widget{
	border:1px solid rgb(149, 149, 149);
}

QWidget#middle2_Widget{
	border:1px solid rgb(149, 149, 149);
}

/*局部样式表bottom*/
QWidget#bottom_Widget{
	border:1px solid rgb(149, 149, 149);
}
QPushButton#add_Btn{
	border-radius:5px;
	background-color: rgb(0, 170, 255);
	font:normal 18px &quot;楷体&quot;;
	color:rgb(255, 255, 255);
}

QPushButton#add_Btn:hover{
	background-color: rgb(0, 85, 255);
}

QPushButton#clos_Widget{
	border-radius:5px;
	background-color: rgb(0, 170, 255);
	font:normal 18px &quot;楷体&quot;;
	color:rgb(255, 255, 255);
}

QPushButton#clos_Widget:hover{
	background-color: rgb(0, 85, 255);
}
</string>
  </property>
  <widget class="QWidget" name="top_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>100</height>
    </size>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="add_Friend">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>添加好友</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="add_Group">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>280</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>50</height>
     </size>
    </property>
    <property name="text">
     <string>添加群聊</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="middle_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>700</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>144</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="info_label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="ok_Btn">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>144</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="middle2_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>700</width>
     <height>220</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>220</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>220</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>5</y>
      <width>670</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>670</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>670</width>
      <height>200</height>
     </size>
    </property>
    <property name="title">
     <string>搜索到的信息</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="info_Widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="bottom_Widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>80</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>45</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="add_Btn">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clos_Widget">
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>46</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="back_Img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>back_Img</zorder>
  <zorder>top_widget</zorder>
  <zorder>middle_Widget</zorder>
  <zorder>middle2_Widget</zorder>
  <zorder>bottom_Widget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
